### PR TITLE
setup: Update links and udev rules

### PR DESCRIPTION
**Pull request by arizaini**
* As a bonus remove duplicate schedtool package left behind

Signed-off-by: arizaini <famegalaxy614@gmail.com>

### CODE REVIEW

**Comment by arizaini** [comment]
@MSF-Jarvis

File: setup/android_build_env.sh [hunk 1]
```diff
@@ -29,15 +29,22 @@ libc6-dev libcap-dev libcloog-isl-dev libexpat1-dev libgmp-dev liblz4-* liblzma*
...
+# Purge problematic packages found in things like Mint 19 and Ubuntu 18
 sudo apt purge -y openjdk-11-*
 
 if [[ ! "$(command -v adb)" == "" ]]; then
     echo -e "Setting up udev rules for adb!"
     sudo curl --create-dirs -L -o /etc/udev/rules.d/51-android.rules -O -L https://raw.githubusercontent.com/M0Rf30/android-udev-rules/master/51-android.rules
-    sudo chmod 644 /etc/udev/rules.d/51-android.rules
-    sudo chown root /etc/udev/rules.d/51-android.rules
-    sudo systemctl restart udev
+    sudo groupdel adbusers
```
Comment (MSF-Jarvis) [inline setup/android_build_env.sh]: Only delete when not required to add it back?

File: setup/android_build_env.sh [hunk 2]
```diff
@@ -29,15 +29,22 @@ libc6-dev libcap-dev libcloog-isl-dev libexpat1-dev libgmp-dev liblz4-* liblzma*
...
+    sudo curl --create-dirs -L -o /usr/lib/sysusers.d/android-udev.conf -O -L https://raw.githubusercontent.com/M0Rf30/android-udev-rules/master/android-udev.conf
```
Comment (MSF-Jarvis) [inline setup/android_build_env.sh]: Only run this when using `systemd-sysusers`?

File: setup/android_build_env.sh [hunk 3]
```diff
@@ -29,15 +29,22 @@ libc6-dev libcap-dev libcloog-isl-dev libexpat1-dev libgmp-dev liblz4-* liblzma*
...
+    if [[ "${LSB_RELEASE}" =~ "Mint 18" || "${LSB_RELEASE}" =~ "Ubuntu 16" ]]; then
+        sudo groupadd adbusers
+    else
+        sudo systemd-sysusers
+    fi
+    sudo usermod -a -G adbusers "$(whoami)"
```
Comment (MSF-Jarvis) [inline setup/android_build_env.sh]: Will the group always exist?

**Comment by MSF-Jarvis** [comment]
Stale

**Pull request closed without merging**